.github/workflows/push-tags.yml

```diff
@@ -35,3 +35,13 @@ jobs:
                        --tag=${{ steps.get_version.outputs.VERSION  }}
       env:
         PROJECT_ID: ${{ secrets.PROJECT_ID }}
+    - name: Push Cloud Shell Image to GCR
+      timeout-minutes: 20
+      run: |
+        set -x
+        gcloud auth configure-docker --quiet
+        IMAGE_PATH=gcr.io/$PROJECT_ID/cloudshell-image:${{ steps.get_version.outputs.VERSION }}
```

Comment:
In this line, what does steps.get_version.outputs.VERSION correspond to?

Reply:
It's a reference to other place in this yaml doc. In this case we want to tag a cloudshell image with the same version tag as the result of get_version step (part of the same workflow).